src/settings_gui.cpp

```diff
@@ -75,24 +75,19 @@ static uint GetCurrentResolutionIndex()
 static void ShowCustCurrency();
 
 template <class T>
-static DropDownList BuildSetDropDownList(int *selected_index, bool allow_selection)
+DropDownList BuildSetDropDownList(int *selected_index)
```

Comment:
Maybe use a reference and not a pointer here? The code seems to assume it's never `nullptr`.
(But it might be a rabbit hole)